src/components/EmptyState/README.md

```diff
@@ -163,6 +163,29 @@ Use to explain a single feature before merchants have used it.
 </EmptyState>
 ```
 
+### Empty state with footer
+
+Use to explain a single feature with additional context before merchants have used it.
+
+```jsx
+<EmptyState
+  heading="Manage your inventory transfers"
+  action={{content: 'Add transfer'}}
+  secondaryAction={{content: 'Learn more', url: 'https://help.shopify.com'}}
+  footer={
+    <p>
+      Import inventory from
+      <Link monochrome url="/settings">
+        settings
+      </Link>
+    </p>
```

Comment:
@biggeidea hey Ryan, this is what will be displayed in the example that will look like this:

<img width="798" alt="Screen Shot 2019-03-15 at 2 07 00 PM" src="https://user-images.githubusercontent.com/6495272/54452465-a2159500-472b-11e9-8ecc-de047f1366bc.png">

Reply:
Here are some suggestions for language:
Empty state footer
Subdued empty state footer
Empty state footer suggestion
Subdued empty state suggestion

Reply:
@biggeidea what about `Import inventory from settings`? How can we improve this copy to illustrate `footer` usage in the context of transfers empty state?

Reply:
Something like this?

If you don't want to add a transfer, you can import your inventory from settings.